playbook/app/pb_kits/playbook/pb_background/_background.jsx

```diff
@@ -62,6 +65,7 @@ const Background = (props: BackgroundProps) => {
             {...dataProps}
             className={classes}
             id={id}
+            style={{ backgroundColor: customColor }}
```

Comment:
Hmm.. I think you are setting a prop to a prop. BackgroundColor is a prop, as well as a customColor.